lang/cs/Org.Apache.REEF.IO/FileSystem/AzureBlob/AzureCloudBlobClient.cs

```diff
@@ -69,6 +69,27 @@ public ICloudBlockBlob GetBlockBlobReference(Uri uri)
         }
 
         public BlobResultSegment ListBlobsSegmented(
+            string prefix,
+            bool useFlatListing,
+            BlobListingDetails blobListingDetails,
+            int? maxResults,
+            BlobContinuationToken continuationToken,
+            BlobRequestOptions blobRequestOptions,
+            OperationContext operationContext)
+        {
+            var task = _client.ListBlobsSegmentedAsync(
+                prefix,
+                useFlatListing,
+                blobListingDetails,
+                maxResults,
+                continuationToken,
+                blobRequestOptions,
+                operationContext);
+            task.Wait();
```

Comment:
this can be task.Result

Reply:
this has been taken care of in a previous PR but I have changed it here too